posts/archive/ansible-servicenow-howto-part-3-making-outbound-restful-api-calls-to-ansible-tower.md

```diff
@@ -0,0 +1,350 @@
+---
+author: Michael Ford
+date: 2019-10-09 00:00 UTC
+description: Making outbound RESTful API calls to Red Hat Ansible Tower
+lang: en-us
+title: Ansible and ServiceNow Part 3, Making outbound RESTful API calls to Red Hat Ansible Tower
+---
+
+# Ansible and ServiceNow Part 3, Making outbound RESTful API calls to Red Hat Ansible Tower
+
+Red Hat Ansible Tower offers value by allowing automation to scale in a
+checked manner - users can run playbooks for only the processes and
+targets they need access to, and no further. 
+
+Not only does Ansible Tower provide automation at scale, but it also
+integrates with several external platforms. In many cases, this means
+that users can use the interface they are accustomed to while launching
+Ansible Tower templates in the background. 
+
+One of the most ubiquitous self service platforms in use today is
+ServiceNow, and many of the enterprise conversations had with Ansible
+Tower customers focus on ServiceNow integration. With this in mind, this
+blog entry walks through the steps to set up your ServiceNow instance to
+make outbound RESTful API calls into Ansible Tower, using OAuth2
+authentication.
+
+The following software versions are used:
+
+-   Ansible Tower: 3.4, 3.5
+-   ServiceNow: London, Madrid
+
+If you sign up for a ServiceNow Developer account, ServiceNow offers a
+free instance that can be used for replicating and testing this
+functionality. Your ServiceNow instance needs to be able to reach your
+Ansible Tower instance. Additionally, you can visit
+<https://ansible.com/license> to obtain a trial license for Ansible
+Tower. Instructions for installing Ansible Tower can be found
+[here](https://docs.ansible.com/ansible-tower/latest/html/quickinstall/prepare.html). 
+
+## Preparing Ansible Tower
+
+1. In Ansible Tower, navigate to **Applications** on the left side of
+the screen. Click the **green plus button** on the right, which will
+present you with a Create Application dialog screen. Fill in the
+following fields:
+
+-   Name: Descriptive name of the application that will contact Ansible
+    Tower
+-   Organization: The organization you wish this application to be a
+    part of
+-   Authorization Grant Type: Authorization code
+-   Redirect URIS:
+    `https://\<snow_instance_id\>.service-now.com/oauth_redirect.do`
+-   Client Type: Confidential
+
+![image3-4](https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=1913&name=image3-4.png){style="width: 1913px;"
+width="1913"
+srcset="https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=957&name=image3-4.png 957w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=1913&name=image3-4.png 1913w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=2870&name=image3-4.png 2870w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=3826&name=image3-4.png 3826w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=4783&name=image3-4.png 4783w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=5739&name=image3-4.png 5739w"
```

Comment:
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.